app/dashboard/templates/bounty_details.html

```diff
@@ -314,6 +314,7 @@ <h5 class="bounty-heading">{% trans "Funder" %}</h5>
   <script src="{% static "v2/js/tokens.js" %}"></script>
   <script src="{% static "v2/js/amounts.js" %}"></script>
   <script src="{% static "v2/js/pages/bounty_details.js" %}"></script>
+  <script src="{% static "v2/js/purify.min.js" %}"></script>
```

Comment:
i dont like it being imported multiple times but i bet it's some kind of optimization:
"get it only when it's really needed"